### PR TITLE
Move counter box to the right of the search button

### PR DESCRIPTION
**Pull request by JukMR**
Remove the counter box from the prompt text box because it was being display over the written text. 

This way we can see clearly what the user write and the counter box is in a more comfortable place

### CODE REVIEW

File: icons/counter.png
Comment (qequ) [inline icons/counter.png:1]: why are you uploading the png again (already merger in a previos pr) MRJUK?

File: manifest.json [hunk 1]
```diff
@@ -1,18 +1,30 @@
 {
-
   "description": "Add a counter to the chat GPT chat window",
   "manifest_version": 2,
   "name": "chatgpt-counter",
   "version": "1.0",
   "icons": {
-    "48": "icons/border-48.png"
+    "48": "icons/counter.png"
```
Comment (qequ) [inline manifest.json]: already updated (?

**Review by qequ — CHANGES_REQUESTED** [review]

File: manifest.json (hunk 1 above)
Comment (JukMR) [reply manifest.json]: Sorry, forgot to update the PR after merging #1. Will fix it now

**Comment by JukMR** [comment]
Had some problems with the configuration. Will create a new PR for this issue.

**Pull request closed without merging**